crates/huub/src/lib.rs

```diff
@@ -1104,7 +1104,13 @@ impl Model {
 		let any_slv: &mut dyn Any = slv.oracle.solver_mut();
 		if let Some(r) = any_slv.downcast_mut::<Cadical>() {
 			r.set_option("restart", config.restart() as i32);
+			r.set_limit("preprocessing", config.preprocessing() as i32);
```

Comment:
Interesting that the limit is set using a `bool`. I would have thought that would be a limit in the sense of the “number of times” something is done or the “amount of time” spent on this.